### PR TITLE
sync: AURURI variable

### PR DESCRIPTION
**Pull request by AladW**
Usually I would update the documentation, but there's none present for `aur-fetch` anyway. (@Foxboron)

### CODE REVIEW

File: lib/aur-fetch [hunk 1]
```diff
@@ -1,67 +1,58 @@
...
-            git clone "$remote/$REPLY".git
+            git clone "$REPLY"
         fi
     done
 }
 readonly -f clone
 
 usage() {
-    printf '%s\n' "usage: $argv0 [-L logdir] [-t]" >&2
+    printf '%s\n' "usage: $argv0 [-L log_dir] [-t]" >&2
```
Comment (Foxboron) [inline lib/aur-fetch]: `printf '%s\n' "usage: $argv0 [-L log_dir] [-t mode]" >&2`

File: lib/aur-fetch [hunk 2]
```diff
@@ -1,67 +1,58 @@
...
 
-while getopts :L:Pt OPT; do
+while getopts :L:t OPT; do
     case $OPT in
-        L) logdir=$OPTARG ;;
-        P) default_path=0 ;;
-        t) snapshot=1     ;;
-        *) usage          ;;
+        L) log_dir=$OPTARG ;;
+        t) mode=tar        ;;
```
Comment (Foxboron) [inline lib/aur-fetch]: This probably needs to be `$OPTARG`?

File: lib/aur-fetch (hunk 1 above)
Comment (AladW) [reply lib/aur-fetch]: The idea was that the default "mode" is `git`, which is then set to `tar` if you specify the `-t` flag. Should be less typing and matches the old `aur-fetch` interface. I guess the initial mode declaration can be moved from the top of the script to getopt to make this more clear.

Comment (Foxboron) [reply lib/aur-fetch]: Right, this makes sense. I'll update the documentation accordingly

**Pull request merged**